Triage/triage_modules/trigger_triage.py

```diff
@@ -125,18 +125,21 @@ def get_malware_samples(self, malware: str, period: int) -> list[str]:
     def check_sample_signature(self, sample_id: str) -> bool:
         try:
             # Get static report for PE metadata
+            signatures = [False]
             static_report = self.client.static_report(sample_id=sample_id)
-            signers = static_report.get("files")[0]["metadata"]["pe"]["code_sign"]["signers"]
-            for value in signers:
-                if isinstance(value["validity"]["trusted"], list):
-                    for entry in value["validity"]["trusted"]:
-                        if entry:
-                            signature = entry
-                else:
-                    signature = value["validity"]["trusted"]
-            return signature
+            for f in static_report.get("files"):
```

Comment:
NoneType is not iterable.
```suggestion
            for f in static_report.get("files", []):
```